start_now: 8
start_togo: 11
oriParam: 0
tolerence: 13.0
decelerationZone: 30.0
nodeLoseConpDELAY: 30
secondCapt: 290
thirdCapt: 400
after_6_shift: 40
time_6: 2
time_7: 2
# time_6: 30
# time_7: 15